quinn-proto/src/connection/datagrams.rs

```diff
@@ -19,6 +19,10 @@ pub struct Datagrams<'a> {
 impl<'a> Datagrams<'a> {
     /// Queue an unreliable, unordered datagram for immediate transmission
     ///
+    /// If `drop` is true, previously queued datagrams which are still unsent will be discarded to
+    /// make space for this datagram. If `drop` is false, and there isn't enough space due to
+    /// previously queued datagrams, this function will return `SendDatagramError::Blocked`
```

Comment:
s/will/may/. We should also be explicit that we drop the oldest datagrams first.